### DISCUSSION TITLE
What would have to be true for us to runa  $25k round on dGrants during GR12?

### DISCUSSION THREAD

**Discussion by owocki** [dcgtc/dgrants#312] [Ideas]
The title says it all "What would have to be true for us to runa  $25k round on dGrants during GR12?"

We'd love to run the "Gitcoin building Gitcoin" round during GR12 on dGrants and show off the tech/progress

**Comment by phutchins** [comment]
If we're running it on polygon then:
1. QA/testing and some bug fixing iterations on the dGrants
2. Some work and planning on the curation side (can be manual, just need a process and people)
3. Some work and planning around our sybil resistance approach (we could potentially leverage the current approach from cGrants or aim for a step or two better)
4. Clarity around what grants would be in the round, how they get there, and the path from gitcoin to cGrants or dGrants
